apis/core/lib/domain/identifiers.ts

```diff
@@ -10,6 +9,6 @@ export function cubeProject(label: string): NamedNode {
   return $rdf.namedNode(`${env.API_CORE_BASE}cube-project/${url.slugify(label)}`)
 }
 
-export function csvMapping(project: GraphPointer<NamedNode>): NamedNode {
-  return $rdf.namedNode(`${project.value}/csv-mapping`)
+export function csvMapping(filename: string): NamedNode {
+  return $rdf.namedNode(`${env.API_CORE_BASE}csv-mapping/${url.slugify(filename)}`)
```

Comment:
This change breaks `createProject` line 23.

The idea was that a project is created first and then the CSV Mapping resource to "dependent" on it. Hence the `project/FOO/csv-mapping` hierarchical URI. You would then upload the file to the mapping resource to create `project/FOO/csv-mapping/${filename}`.

Reply:
Maybe we could handle project creation by CSV upload and create the project based off the CSV name?

| What | URI |
| -- | -- |
| File | `cube data.csv` |
| Project | `/project/cube-data-slug` |
| Mapping | `/project/cube-data-slug/csv-mapping` |
| Source | `/project/cube-data-slug/csv-mapping/cube-data.csv` |

All those would have to be created when a project is created from a CSV directly. @martinmaillard ?

Reply:
It's definitely something we need to discuss to move forward with CSV uploads.

Reply:
I think it would be good to have a discussion about the data model and workflow as well how we store the the data on a S3 bucket. I will amend the corresponding issue #13